packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -726,12 +727,21 @@ export class FeatureDevController {
                 return
             }
         } catch (err: any) {
-            this.messenger.sendErrorMessage(
-                createUserFacingErrorMessage(err.message),
-                message.tabID,
-                this.retriesRemaining(session),
-                session?.state.phase
-            )
+            if (err instanceof WorkspaceFolderNotFoundError) {
+                this.messenger.sendAnswer({
```

Comment:
how is this change related?

Reply:
This has been updated based on UX feedback [here](https://quip-amazon.com/fA3JAlEEK3ab/QFeatureDev-Errors-string-review#temp:C:SAH3f48f3e68c914276b0c7517c6) as the older message contained feature name which was required to be removed.

Reply:
Is the WorkspaceFolderNotFoundError actually possible here?

Reply:
Yes, this is where `WorkspaceFolderNotFoundError` will be thrown for case when workspace is not open.

Reply:
I think is a valid remark then the one from justin, it feels like it is being combined two features into one. The rename and the handling of this new error. I approve anyway

Reply:
This was changed as we do not want to show feature name in case of this error message.